### PR TITLE
{lib}[GCCcore/9.3.0] ACTC v1.1

### PR DESCRIPTION
**Pull request by ekieffer**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ekieffer** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-002 - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/42b8dd7f7894b579fc9d27bdac8534ff for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node377.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ad032e8a193a12f923ec586ea0500d3b for a full test report.

File: easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'ACTC'
+version = '1.1'
+
+homepage = 'https://sourceforge.net/projects/actc'
+description = "ACTC converts independent triangles into triangle strips or fans."
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
```
Comment (lexming) [inline easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-9.3.0.eb:9]: The `toolchainopts` from `ACTC-1.1-GCCcore-8.3.0.eb` are missing
```suggestion
toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
toolchainopts = {'pic': True}
```

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node378.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/9eb4fc767f4bf4aeed8f596b11ff9c70 for a full test report.

**Comment by smoors** [comment]
@boegelbot please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @ekieffer!

**Pull request merged**